.changeset/sixty-boats-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Add at-rule-no-vendor-prefix support computing EditInfo.
```

Comment:
```suggestion
Added: `at-rule-no-vendor-prefix` support computing `EditInfo`
```

See: https://github.com/stylelint/stylelint/blob/main/docs/maintainer-guide/pull-requests.md#merging

For the changeset we use the past tense and a colon.
While for the pull requests and issues we use the present tense.